docs/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance.cs

```diff
@@ -118,11 +76,162 @@ public static void PFI_Regression()
             }
         }
 
+        public static void PFI_BinaryClassification()
+        {
```

Comment:
Don't you need to reference this new sample in the constructors?

Reply:
They refer to the file, but not the line or function name: 
`~/../docs/samples/doc/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance.cs`